jbpm-wb-process-runtime/jbpm-wb-process-runtime-backend/src/main/java/org/jbpm/workbench/pr/backend/server/RemoteProcessRuntimeDataServiceImpl.java

```diff
@@ -149,7 +149,7 @@ public ProcessInstanceDiagramSummary getProcessInstanceDiagramSummary(ProcessIns
         summary.setSubProcessInstances(getProcessInstancesByParentId(processInstanceKey, processInstanceKey.getDeploymentId()));
 
         if (processInstance.getParentId() != -1) {
-            summary.setParentProcessInstanceSummary(getProcessInstance(new ProcessInstanceKey(processInstanceKey.getServerTemplateId(), "", processInstance.getParentId())));
+            summary.setParentProcessInstanceSummary(getProcessInstance(new ProcessInstanceKey(processInstanceKey.getServerTemplateId(), processInstanceKey.getDeploymentId(), processInstance.getParentId())));
```

Comment:
@bxf12315 Could you please add some testing here?

Reply:
Done